src/server/rpc/procedures/azure-translation/azure-translation.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Uses Microsoft's Azure Cognitive Services API to translate text.
+ * For more information, check out https://azure.microsoft.com/en-us/pricing/details/cognitive-services/translator-text-api/.
+ *
+ * Terms of use: https://www.microsoft.com/en-us/servicesagreement
+ * @service
+ */
+
+const ApiConsumer = require('../utils/api-consumer');
+const TranslationConsumer = new ApiConsumer('azure-translation', 'https://api.cognitive.microsofttranslator.com/',{cache: {ttl: 15 * 60}});
```

Comment:
I think we could increase this to higher values

Reply:
Once we have better caching, we should do that. Right now caching is only really working for the languages option.